### DISCUSSION TITLE
wav audio transcription problem

### DISCUSSION THREAD

**Discussion by Toasterme** [open-webui/open-webui#2920] [General]
# Bug Report

## Description
I have switched the language to french and the audio to Google French and i have an error message, it cannot speak french.

**Bug Summary:**
[Provide a brief but clear summary of the bug]
Trying the audio language and i have this error : 
Something went wrong :/ [Errno 1094995529] Invalid data found when processing input: '/app/backend/data/cache/audio/transcriptions/bcc8ab39-b93e-4a3a-b413-13d6a7fa8f2f.wav'

**Steps to Reproduce:**
switch language of the webui to french, and under Audio:
Default (whisper)
Autosend: activated
Api web
disabled
Google french 
Allow non-local voices :activated 

**Expected Behavior:**
i expect a french answer

**Actual Behavior:**
error message like i said and it answer something random in english

## Environment

- **Open WebUI Version:** v0.2.5
- **Ollama (if applicable):** 0.1.42

- **Operating System:** Ubuntu 23.10
- **Browser (if applicable):** Chrome

## Reproduction Details

**Confirmation:**

- [x] I have read and followed all the instructions provided in the README.md.
- [x] I am on the latest version of both Open WebUI and Ollama.
- [x] I have included the browser console logs.
- [x] I have included the Docker container logs.

## Logs and Screenshots

**Browser Console Logs:**
n/a

**Docker Container Logs:**
Something went wrong :/ [Errno 1094995529] Invalid data found when processing input: '/app/backend/data/cache/audio/transcriptions/bcc8ab39-b93e-4a3a-b413-13d6a7fa8f2f.wav'

**Screenshots (if applicable):**
[Attach any relevant screenshots to help illustrate the issue]

## Installation Method

docker, so i have docker compose for just ollama and webui

## Additional Information

n/a

## Note

If the bug report is incomplete or does not follow the provided instructions, it may not be addressed. Please ensure that you have followed the steps outlined in the README.md and troubleshooting.md documents, and provide all necessary information for us to reproduce and address the issue. Thank you!

**Comment by daaain** [comment]
I also keep getting this error all the time even when not speaking French...

```
Something went wrong :/ [Errno 1094995529] Invalid data found when processing input: '/app/backend/data/cache/audio/transcriptions/75f8edac-8258-4b95-ac38-f9c0a8033820.wav'
```

It also seems to feed back the output audio as input? Maybe that's the issue?

<img width="1047" alt="image" src="https://github.com/open-webui/open-webui/assets/69962/14e3a9f4-c375-4446-b24a-5018fb4ad6a7">

Not sure if matters, but using this Docker command:

```sh
docker run --rm -p 3000:8080 --add-host host.docker.internal=host-gateway -e WEBUI_AUTH=false -e OPENAI_API_BASE_URL=http://host.docker.internal:1234/v1 -e OPENAI_API_KEY=lm-studio -v open-webui:/app/backend/data --name open-webui ghcr.io/open-webui/open-webui:main
```

Realised that there is a stack trace in the logs:

```
INFO:apps.audio.main:file.content_type: audio/wav
244464d0-c026-4eab-943c-26669a1d434e.wav
ERROR:apps.audio.main:[Errno 1094995529] Invalid data found when processing input: '/app/backend/data/cache/audio/transcriptions/244464d0-c026-4eab-943c-26669a1d434e.wav'
Traceback (most recent call last):
  File "/app/backend/apps/audio/main.py", line 299, in transcribe
    segments, info = model.transcribe(file_path, beam_size=5)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/faster_whisper/transcribe.py", line 318, in transcribe
    audio = decode_audio(audio, sampling_rate=sampling_rate)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/faster_whisper/audio.py", line 46, in decode_audio
    with av.open(input_file, mode="r", metadata_errors="ignore") as container:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "av/container/core.pyx", line 420, in av.container.core.open
  File "av/container/core.pyx", line 266, in av.container.core.Container.__cinit__
  File "av/container/core.pyx", line 286, in av.container.core.Container.err_check
  File "av/error.pyx", line 328, in av.error.err_check
av.error.InvalidDataError: [Errno 1094995529] Invalid data found when processing input: '/app/backend/data/cache/audio/transcriptions/244464d0-c026-4eab-943c-26669a1d434e.wav'
INFO:     192.168.65.1:33736 - "POST /audio/api/v1/transcriptions HTTP/1.1" 400 Bad Request
['ur13Y0flQlskhZ3AAAAD']
```
